src/mappers/provinces/province_mapper_importer.cpp

```diff
@@ -80,23 +118,34 @@ void CheckAllHoi4ProvincesMapped(const mappers::Hoi4ToVic3ProvinceMapping& hoi4_
 
 }  // namespace
 
-
-
-mappers::ProvinceMapperImporter::ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem)
+mappers::ProvinceMapperImporter::ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem,
+    const std::optional<vic3::World>& vic3_world)
 {
    version_parser_.registerRegex(R"(\d\.[\d]+\.\d)",
        [this, &filesystem](const std::string& unused, std::istream& input_stream) {
           mapping_parser_.parseStream(input_stream);
           CheckAllHoi4ProvincesMapped(hoi4_to_vic3_province_map_, filesystem);
        });
 
-   mapping_parser_.registerKeyword("link", [this](std::istream& input_stream) {
+   mapping_parser_.registerKeyword("link", [this, &vic3_world](std::istream& input_stream) {
```

Comment:
```suggestion
   mapping_parser_.registerKeyword("link", [this, vic3_world](std::istream& input_stream) {
```

Whenever possible, capture by value instead of by reference to ensure const-ness